.github/workflows/release-v2.yml

```diff
@@ -37,14 +37,14 @@ jobs:
 
       - name: Set reference
         id: vars
-        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+        run: echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
```

Comment:
a few of these are changes we had made to the release v2 workflow on the main branch only. porting to v1 branch for consistency.